### DISCUSSION TITLE
Github HousewareHQ Octernships

### DISCUSSION THREAD

**Discussion by krcpr007** [community/community#49901] [GitHub Education]
Hey, I have completed the Houseware Octernships task but don't have any idea how to send it or submit it for review.
Can anyone please help me?

**Comment by maahaMaya** [comment]
[You can follow this channel.](https://www.youtube.com/watch?v=Rx8myzgSTg8)
https://www.youtube.com/watch?v=Rx8myzgSTg8

All the details are there.

**Reply by krcpr007** [reply]
This is not a solution in this video he is just telling about Github Octernship. Not solving my issue

**Reply by AbhijithGanesh** [reply]
Please delete this comment, it is not useful and misleading!

**Comment by bkpecho** [comment]
Hello brother @krcpr007 

Great job on completing the Houseware Octernships task! To submit your work for review, you can follow these steps:

### Task submission
Students are expected to use the [GitHub Flow](https://docs.github.com/en/get-started/quickstart/github-flow) when working on their project.

1. Making changes on the auto generated feedback branch to complete the task
2. Using the auto generated Feedback Pull Request for review and submission
3. Using GitHub Discussions to ask any relevant questions regarding the project

If you have any questions or issues with the above steps, feel free to check this [link](https://github.com/HousewareHQ/backend-engineering-octernship#task-submission) or reach out to the project maintainer for assistance. 

Hope my answer helps. Good luck with your submission!

**Reply by krcpr007** [reply]
So i have to done my changes in the feedback branch?

**Reply by bkpecho** [reply]
Yes. As Naveen mentioned in this [comment](https://github.com/HousewareHQ/frontend-engineering-octernship/issues/6#issue-1620071207), he has made his commits in the feedback branch.

**Reply by krcpr007** [reply]
It's not about who is doing how. it's all about the right way...
That's why I am asking I just have do code in feedback branch and just push it ?

**Reply by bkpecho** [reply]
Yes, use the feedback branch. As stated in [instruction number 1](https://github.com/HousewareHQ/backend-engineering-octernship#task-submission), you should push your changes to the `feedback` branch to complete the task.

**Reply by krcpr007** [reply]
![image](https://user-images.githubusercontent.com/80947662/224681904-a763a714-e996-4392-8e06-753a52abcb6c.png)

Its showing like this and all the changes I have done in the main branch its showing below is it correct?
Or i have to push to code feedback branch?

**Reply by bkpecho** [reply]
That's the **auto-generated Feedback Pull Request**. Yes, you may now push that 👍 Congrats on completing the Houseware Octernships task.  Good luck with your submission!

**Reply by krcpr007** [reply]
I'm confused
Now I don't have to do anything?
My submission is done?

**Reply by bkpecho** [reply]
@krcpr007 Have you already created a pull request?

**Reply by krcpr007** [reply]
NO. Its auto generated

**Comment by Sachinsharmak** [comment]
can i use the gin-gonic for this project ?

**Reply by krcpr007** [reply]
No

**Comment by nitishsh1** [comment]
can you tell me how to submit the project?

**Reply by AbhijithGanesh** [reply]
You need to commit files to the main branch, ideally through a PR
